bee-message/src/output/native_token.rs

```diff
@@ -64,12 +64,11 @@ impl NativeTokens {
     /// Maximum possible number of different native tokens that can reside in one output.
     pub const COUNT_MAX: u16 = 256;
 
-    /// Creates a new `NativeTokens`.
+    /// Creates a new [`NativeTokens`].
```

Comment:
```suggestion
    /// Creates new [`NativeToken`]s.
```

Reply:
But we are actually creating a `NativeTokens`, not `NativeToken`s

Reply:
The _a_ and the plural are throwing me off. How about:

```suggestion
    /// Creates a new [`NativeTokens`] object.
```

Reply:
I can add it for now if you want but it wouldn't be very consistent with the rest, we never put `object` in other docs and it's not the only plural form...

Reply:
![image](https://user-images.githubusercontent.com/31802960/151561646-f905846f-510c-4eba-9c5e-f35876412cf2.png)

Reply:
Jokes aside, i would use the term `value` instead of `object`.